phantomgames/Search/CustomTableViewCell/CustomTableViewCell.swift

```diff
@@ -33,22 +32,22 @@ class CustomTableViewCell: UITableViewCell {
     func populateWith(game: Game) {
         titleLabel.text = game.title
         genreLabel.text = game.genre
-                if let imageURL = URL(string: game.thumbnail) {
-                    icon.sd_setImage(with: imageURL, placeholderImage: Constants.ImageConstants.placeholder)
-                } else {
-                    icon.image = Constants.ImageConstants.placeholder
-                }
-             if let gameURL = URL(string: game.gameURL) {
-                 self.gameURL = gameURL
-             }
+        if !game.thumbnail.isEmpty {
+            icon.downloaded(from: game.thumbnail)
+        } else {
+            icon.image = Constants.ImageConstants.placeholder
+        }
+        if let gameURL = URL(string: game.gameURL) {
+            self.gameURL = gameURL
+        }
     }
     
     @IBAction func playButtonTapped(_ sender: Any) {
```

Comment:
private please.
Please rad over the standards documents we have given you guys. Please apply changes everywhere else as well :)